charts/ctlog/values.yaml

```diff
@@ -0,0 +1,59 @@
+namespace:
+  create: false
+  name: ctlog-system
+server:
+  replicaCount: 1
+  image:
+    registry: ghcr.io
+    repository: vaikas/ct_server
```

Comment:
We might add a comment / whatever somewhere putting a pointer in readme or something that points to the repo, or explains why we're pulling these from my repo and say it's being upstreamed as much as we can, just takes a bit of time.

Reply:
README could work

Reply:
should we move those to the sigstore or another place and create some automation to publish the images? instead of using a personal repository/registry